### ISSUE TITLE
Any way of being able to scroll the name?  

### ISSUE THREAD

**Issue by VIOrealm** [coldino/Larkator#20]
I use several mods that add dinos.  Most of them have the similar named dinos which they usually differentiate at the end of the name.  For example, Wyvern_Charcter_BP_Brute_C & Wyvern_Charcter_BP_Water_C   I can't see which one is which even if I minimize font to the lowest setting which is 11.  

Also, out of curiosity, which name are you using?  I don't know the actual field names but going off the labels on https://ark.gamepedia.com/Creature_IDs, this program seems to pick up the "name", "dino name tag" or "entity id".  

An option to choose the "name" or "entity id" would be great.  Entity ID would solve this post.

**Comment by coldino** [comment]
It uses Ark's translation file for the species names, by looking up the ClassName. This won't work for modded species, so you just get the ClassName directly. I'm not sure what you mean by Entity ID, but that isn't available.

Shrinking the font further doesn't work well. I could perhaps add a tooltip with the full name as a backup plan.

**Comment by VIOrealm** [comment]
I don't know the name of it within the database but I'm going off the column label from https://ark.gamepedia.com/Creature_IDs.  Sorry but I created another issue before noticing that you responded to my previous comments.  Did you stop using the file classes.json?  I used that to abbreviate or truncate the name to something more logical organized (for my use).  Here's an example of what I did with it.  https://survivetheark.com/index.php?/forums/topic/376809-larkator-backup/ I translated what I believe you say is the ClassName to an abbreviated one with the mod initials up front.  The problem is that I can no longer find the classes.json file to do that.
